test/Microsoft.Azure.ServiceBus.UnitTests/SenderReceiverClientTestBase.cs

```diff
@@ -157,5 +158,60 @@ public abstract class SenderReceiverClientTestBase
 
             var receivedMessages = await TestUtility.ReceiveMessagesAsync(messageReceiver, messageCount);
         }
+
+        protected async Task ScheduleMessagesAsyncTestCase(MessageSender messageSender, MessageReceiver messageReceiver, int messageCount)
```

Comment:
Would be really nice to see a test name saying "scheduled message appears on or after the time it was schedule for".